### PR TITLE
Cleanup batch side from specific code for devcockpit and views

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: sonar-batch/src/main/java/org/sonar/batch/cpd/JavaCpdEngine.java [hunk 1]
```diff
@@ -60,17 +59,10 @@
...
 import java.util.Set;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import java.util.concurrent.*;
 
 public class JavaCpdEngine extends CpdEngine {
```
Comment (simonbrandhof) [inline sonar-batch/src/main/java/org/sonar/batch/cpd/JavaCpdEngine.java]: what's the link between these changes and views/cockpit cleanup ?

File: sonar-batch/src/main/java/org/sonar/batch/mediumtest/TaskResult.java [hunk 2]
```diff
@@ -115,19 +114,24 @@ public BatchReportReader getReportReader() {
   }
 
   private void storeMeasures(ProjectScanContainer container) {
-    InputPathCache inputFileCache = container.getComponentByType(InputPathCache.class);
+    ResourceCache resourceCache = container.getComponentByType(ResourceCache.class);
```
Comment (simonbrandhof) [inline sonar-batch/src/main/java/org/sonar/batch/mediumtest/TaskResult.java]: is there a link with views and cockpit here ?

**Comment by simonbrandhof** [comment]
not sure that everything relates to views/cockpit, but  LGTM

File: sonar-batch/src/main/java/org/sonar/batch/mediumtest/TaskResult.java (hunk 2 above)
Comment (henryju) [inline sonar-batch/src/main/java/org/sonar/batch/mediumtest/TaskResult.java]: No. I also updated some medium tests and needed this change.

**Pull request merged**